runners/core-java/src/main/java/org/apache/beam/runners/core/SideInputHandler.java

```diff
@@ -163,11 +164,6 @@ public void addSideInputValue(
   @Override
   public <T> T get(PCollectionView<T> sideInput, BoundedWindow window) {
 
-    if (!isReady(sideInput, window)) {
```

Comment:
Why is this removed? Elements should not be processed until all of their side inputs are ready.